Source/Core/DolphinQt2/GameList/TableDelegate.cpp

```diff
@@ -28,18 +30,22 @@ void TableDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
     DrawPixmap(painter, option.rect, Resources::GetCountry(data.toInt()));
     break;
   case GameListModel::COL_RATING:
-    DrawPixmap(painter, option.rect, Resources::GetRating(data.toInt()));
+    DrawPixmap(painter, option.rect, Resources::GetRating(data.toInt()), false);
     break;
   case GameListModel::COL_BANNER:
     DrawPixmap(painter, option.rect,
-               data.value<QPixmap>().scaled(NORMAL_BANNER_SIZE, Qt::KeepAspectRatio,
-                                            Qt::SmoothTransformation));
+               data.value<QPixmap>().scaled(NORMAL_BANNER_SIZE * scale, Qt::KeepAspectRatio,
+                                            Qt::SmoothTransformation),
+               NORMAL_BANNER_SIZE * scale);
     break;
   case GameListModel::COL_SIZE:
     painter->drawText(option.rect, Qt::AlignCenter, FormatSize(data.toULongLong()));
     break;
-  // Fall through.
   case GameListModel::COL_ID:
+    painter->drawText(option.rect,
```

Comment:
Yeah, that is preferable but I wasn't sure what the right angle for doing that should be. Just adding a fixed value? Sounds kinda wrong to me. But I'm open to any suggestions.